oci/pull.bzl

```diff
@@ -516,6 +463,7 @@ pin_tag = repository_rule(
         "image": attr.string(doc = "The name of the image we are fetching, e.g. `gcr.io/distroless/static`", mandatory = True),
         "tag": attr.string(doc = "The tag being used, e.g. `latest`", mandatory = True),
         "toolchain_name": attr.string(default = "oci", doc = "Value of name attribute to the oci_register_toolchains call in the workspace."),
```

Comment:
nit: seems like it's time to extract a dict constant for the attrs that are common between alias_target and the one above